tests/periph_i2c/tests/test.py

```diff
@@ -0,0 +1,328 @@
+    t.run_test(i2c.write_reg(d_num, __BPT_ADDR-4, __BPT_USER_REG, 0), "Error")
+
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG, [44])
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG, [0])
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG+1, [45, 46])
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG+1, [1, 2])
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG, [47, 48, 49])
+    check_write_regs(t, d_num, __BPT_ADDR, __BPT_USER_REG, [0, 1, 2])
+    t.run_test(i2c.write_regs(d_num, __BPT_ADDR-5, __BPT_USER_REG, [0]),
+               "Error")
+
+    t.run_test(i2c.write_byte(d_num, __BPT_ADDR, __BPT_USER_REG+1), "Success")
+    t.run_test(i2c.read_byte(d_num, __BPT_ADDR), "Success", [1])
+    t.run_test(i2c.write_byte(d_num, __BPT_ADDR, __BPT_USER_REG), "Success")
+    t.run_test(i2c.read_byte(d_num, __BPT_ADDR), "Success", [0])
+    t.run_test(i2c.write_byte(d_num, __BPT_ADDR+5, 0), "Error")
+
+    t.run_test(i2c.write_bytes(d_num, __BPT_ADDR, [__BPT_USER_REG+1]),
+               "Success")
+    t.run_test(i2c.read_byte(d_num, __BPT_ADDR), "Success", [1])
+    t.run_test(i2c.write_bytes(d_num, __BPT_ADDR, [__BPT_USER_REG, 50]),
+               "Success")
+    t.run_test(i2c.read_byte(d_num, __BPT_ADDR), "Success", [50])
+    t.run_test(i2c.write_bytes(d_num, __BPT_ADDR, [__BPT_USER_REG+1, 51, 52]),
+               "Success")
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 2), "Success", [51, 52])
+    t.run_test(i2c.write_bytes(d_num, __BPT_ADDR, [__BPT_USER_REG, 0, 1, 2]),
+               "Success")
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 3), "Success", [0, 1, 2])
+
+    t.run_test(i2c.write_bytes(d_num, __BPT_ADDR-7, [47, 48, 49]), "Error")
+
+    t.run_test(i2c.release(d_num), "Success")
+    return t
+
+
+def read_flag_test():
+    cmd_log = list()
+    t = Test('read flag test', 'Tests the read flag functionality', cmd_log)
+    t.notes += 'Not a comprehesive test but tests a proper frame\n'
+
+    setup_test(t)
+
+    d_num = 0
+
+    t.run_test(i2c.release(d_num), "Success")
+    t.run_test(i2c.acquire(d_num), "Success")
+    t.run_test(i2c.read_reg(d_num, __BPT_ADDR, __BPT_USER_REG), "Success", [0])
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 4), "Success", [0])
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 12), "Success", [1])
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 8), "Success", [2])
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 8), "Error")
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 12), "Error")
+    t.skip_test("i2c.read_bytes(d_num, __BPT_ADDR, 1, 4), Success, [0]")
+    t.skip_test("i2c.read_bytes(d_num, __BPT_ADDR, 1, 4), Error, [0]")
+    t.skip_test("RESET DUT")
+    t.run_test(i2c.read_bytes(d_num, __BPT_ADDR, 1, 1))
+
+    return t
+
+
+def print_full_result(test):
+    print('===================================================================\
+=============')
```

Comment:
minor, but this looks a bit ugly - can't we live we no line break or less `====` to fit 80 chars?

Reply:
Again throw away so it doesn't matter.